ibis/backends/duckdb/__init__.py

```diff
@@ -61,3 +69,13 @@ def fetch_from_cursor(
     ):
         df = cursor.cursor.fetch_df()
         return schema.apply_to(df)
+
+    def _get_schema_using_query(self, query: str) -> sch.Schema:
```

Comment:
Seems like it will need a rebase after #3656

Reply:
Done.